### PR TITLE
feat: user guide updates

### PR DESCRIPTION
**Pull request by sr-remsha**

### CODE REVIEW

**Comment by sr-remsha** [comment]
update

File: docs/user-guide.md [hunk 1]
```diff
@@ -138,7 +138,7 @@ The section with your prompts is located in the right-hand area of the screen. I
 To create a new prompt:
 
 1.	On the right panel, select **New prompt**.
-2.	Fill in the **Name**, **Description**, and **Prompt** boxes.
+2.	Fill in the **Name**, **Description**, and **Prompt** boxes. The following symbols in the prompt names are prohibited: ":", ";", "/", ",", "=".
```
[the remark this answers is not in the document]
Comment (VolhaBazhkova) [reply docs/user-guide.md:139]: Not resolved here. Please update

File: docs/user-guide.md [hunk 2]
```diff
@@ -192,18 +190,42 @@ You can gather your prompts in folders.
  
 You can create a hierarchy of folders with three nesting levels. Just create a folder and drag-n-drop it in the other folder to create a nesting level. 
 The same way, you can either drag-n-drop a prompt into a folder or use **Move to** in the context menu of a prompt to place it in one of the folders.  
+Empty folders are deleted after refreshing a page or relogining.
+The following symbols in the folders names are prohibited: ":", ";", "/", ",", "=".
```
Comment (VolhaBazhkova) [inline docs/user-guide.md:194]: Please update according to comment above

File: docs/user-guide.md [hunk 3]
```diff
@@ -216,7 +238,7 @@ To create a conversation:
 3.	Optionally, under **System prompts**, select a prompt by typing "/" in the text box. For details, refer to the [Prompts](#prompts) section of this guide.
 4.	Under **Temperature**, set a value of the temperature parameter. For details, refer to the [Temperature](#temperature) section of this guide.
 5.	Optionally, under **Addons**, select an Addon. For details, refer to the [Addons](#addons) section of this guide.
-6.	Under conversation settings, type your text (prompt) in the chat box and click the **Send** icon. The conversation will be created, and your first question will be used as its name. 
+6.	Under conversation settings, type your text (prompt) in the chat box and click the **Send** icon. The conversation will be created, and your first question will be used as its name. The following symbols in the conversation names are prohibited: ":", ";", "/", ",", "=" and will be excluded.
```
Comment (VolhaBazhkova) [inline docs/user-guide.md:241]: { and } are restricted symbols also

File: docs/user-guide.md [hunk 4]
```diff
@@ -242,30 +267,81 @@ You can gather your conversations in folders.
 ![](./img/Move_to.png)
 
 You can create a hierarchy of folders with three nesting levels. Just create a folder and drag-n-drop it in the other folder to create a nesting level. The same way, you can either drag-n-drop a conversation into a folder or use **Move to** in the context menu of a conversation to move it to a folder.
+Empty folders are deleted after refreshing a page.
+
+The following symbols in the folders names are prohibited: ":", ";", "/", ",", "=".
```
Comment (VolhaBazhkova) [inline docs/user-guide.md:272]: { and } are restricted symbols also

**Review by VolhaBazhkova — CHANGES_REQUESTED** [review]

**Review by VolhaBazhkova — APPROVED** [review]

**Pull request merged**